spec/arrays.dd

```diff
@@ -155,7 +155,6 @@ $(H2 $(LNAME2 literals, Array Literals))
 ---
 auto a1 = [1,2,3];  // type is int[], with elements 1, 2 and 3
 auto a2 = [1u,2,3]; // type is uint[], with elements 1u, 2u, and 3u
-int[2] = [1, 2];
```

Comment:
```diff
-int[2] = [1, 2];
+int[2] = [1, 2];  // type is `int[2]`  with elements 1, 2 and always holds two elements
```

Reply:
i guess, but you need to add an identifier name like `a3` in that case (as i said).

Reply:
Whoops that's what i meant to do.

Reply:
I would remove the "always holds two elements" part since that has not really much to do with the "array literal".